documents/motions/08.2020.2 - Trainee Delegates.md

```diff
@@ -0,0 +1,52 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 8.2020.2  
+**Subject:** WCA Trainee Delegates  
+**Intent:** Procedure of acknowledgement of WCA Trainee Delegates and description of their rights and duties  
+**Submitted by:** Board of Directors  
+**Date:** January 31, 2020  
+
+# Motion
+
+Trainee Delegates are Community Members who have been appointed by the WCA to train for the role of WCA Delegate.
+
+1. Trainee Delegates will require a time period of extra mentoring and assistance by Senior Delegates and/or Regional Delegates, to support their development to Candidate Delegates and Staff of the WCA.
+   1. A Senior Delegate may nominate a Trainee Delegate for a promotion to Candidate Delegate, if the Senior Delegate judges that the Trainee Delegate has shown their fitness for the position.
+   2. To promote a Trainee Delegate to Candidate Delegate, the Senior Delegate must submit the corresponding application to the WCA Board.
+   3. The WCA Board rejects or approves the application.
+   4. After approval, the Senior Delegate appoints the person as Candidate Delegate.
+2. Rights of Trainee Delegates
+   1. Trainee Delegates have the right to be listed as a Delegate at WCA Competitions in which there is at least one other WCA Delegate listed.
+	    1. At least one listed Delegate must possess the position of Candidate Delegate, Full Delegate, Regional Delegate, or Senior Delegate.
+   2. Trainee Delegates have the right to make decisions at competitions in which they are listed as a Delegate.
+	 3. Trainee Delegates have the right to read and participate in conversations in the Delegates and Reports groups.
+	 4. Trainee Delegates have the right to contribute to and submit Delegate Reports about competitions that they helped oversee.
```

Comment:
In my opinion, a trainee delegate should have the right to contribute to competition reports, but **not** to submit them. This ensures an overseeing delegate in the loops which enforces their training.

Reply:
Agreed. I suggested accordingly.